nessie/merged/sis_profile.py

```diff
@@ -69,18 +69,18 @@ def merge_sis_profile_academic_status(sis_student_api_feed, sis_profile):
     # The Hub may return multiple academic statuses. We'll prefer an undergraduate enrollment, but
     # otherwise select the first well-formed status that is not a Law enrollment.
     academic_status = None
-    career_code = None
```

Comment:
The old code kept Law enrollments out of `academic_status` but let "LAW" slip through as the `career_code`.